src/Service/VendorService/Boardgamegeek/BoardgamegeekVendorService.php

```diff
@@ -15,6 +15,6 @@ class BoardgamegeekVendorService extends AbstractTsvVendorService
 {
     protected const VENDOR_ID = 11;
 
-    protected string $vendorArchiveDir = 'Boardgamegeek';
+    protected string $vendorArchiveDir = 'BoardGameGeek';
```

Comment:
Why change this folder name?

Reply:
Was in clean up mode again. "BoardGameGeek" is correct name: https://boardgamegeek.com/